test/services/CashTransactionsServiceSpec.scala

```diff
@@ -181,6 +185,90 @@ class CashTransactionsServiceSpec extends SpecBase {
       }
     }
   }
+  "submitCashAccountStatementRequest" should {
+
+    "return Left with ErrorDetail if api request failed" in new Setup {
```

Comment:
Please update to as below

"return Left with ErrorDetail if api request fails"